ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -100,15 +98,17 @@ data CacheDirs = CacheDirs
   { hiCacheDir, hieCacheDir, oCacheDir :: Maybe FilePath}
 
 data SessionLoadingOptions = SessionLoadingOptions
-  { findCradle          :: FilePath -> IO (Maybe FilePath)
-  , loadCradle          :: FilePath -> IO (HieBios.Cradle Void)
+  { findCradle             :: FilePath -> IO (Maybe FilePath)
+  -- | Load the cradle with the given Config. If the config is 'Nothing', use the provided
+  -- project root directory to determine the 'Cradle' type.
```

Comment:
This comment could be easier to understand. What does 'Config' mean in this context?

Reply:
That would be easier to read IMO